mappings/net/minecraft/raid/Raid.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3765 net/minecraft/entity/raid/Raid
```

Comment:
Should raid really go to root package... it is such a small thing

Reply:
maybe `world` or `server.world`

Reply:
maybe `server.raid` package? or village?

Reply:
Well villages are always the subject of a raid, so that makes sense